api/client/info.go

```diff
@@ -151,5 +152,19 @@ func (cli *DockerCli) CmdInfo(args ...string) error {
 	if info.ClusterAdvertise != "" {
 		fmt.Fprintf(cli.out, "Cluster Advertise: %s\n", info.ClusterAdvertise)
 	}
+
+	if info.RegistryConfig != nil && (len(info.RegistryConfig.InsecureRegistryCIDRs) > 0 || len(info.RegistryConfig.IndexConfigs) > 0) {
+		fmt.Fprintln(cli.out, "Insecure registries:")
+		for _, registry := range info.RegistryConfig.IndexConfigs {
+			if registry.Secure == false {
+				fmt.Fprintf(cli.out, " %s\n", registry.Name)
```

Comment:
Minor cosmetic nit: I think this would be simpler as `fmt.Fprintln(cli.out, " "+registry.Name)`. This is mostly personal preference - feel free to ignore.